src/js/controller/controller.js

```diff
@@ -593,6 +601,7 @@ Object.assign(Controller.prototype, {
         }
 
         function _getConfig() {
+            // TODO: merge viewModel config or implement complete config in ViewModel?
```

Comment:
My vote is to merge the ViewModel and not conflate it with the complete config.

Reply:
What I'm saying is should the controller do the merge, or should view-model, since view-model has everything.